.nvmrc

```
@@ -1 +1 @@
-v16
+v18.6.0
```

```suggestion
v18.6
```
I think we can allow patches?